Author: Chris Lamb <lamby@debian.org>
Date:   Wed Apr 11 10:23:16 2018 +0200

    Allow to override build date
    
    in order to make builds reproducible.
    See https://reproducible-builds.org/ for why this is good
    and https://reproducible-builds.org/specs/source-date-epoch/
    for the definition of this variable.
    
    This date call works with GNU and BSD variants of date.
Index: Makefile
===================================================================
--- Makefile.orig
+++ Makefile
@@ -10,7 +10,11 @@
 # The version number, for release:
 
 VER_VERSION:=0.9.22
-VER_DATE:=$(shell date +"%Y-%m-%d")
+ifdef SOURCE_DATE_EPOCH
+       VER_DATE=$(shell date -u -d "@$(SOURCE_DATE_EPOCH)" "+%Y-%m-%d" 2>/dev/null || date -u -r "$(SOURCE_DATE_EPOCH)" "+%Y-%m-%d" 2>/dev/null || date -u "+%Y-%m-%d")
+else
+       VER_DATE=$(shell date "+%Y-%m-%d")
+endif
 MAGIC_API_VERSION:=0x00000003
 
 # Need to know the OS
